gems/spree/OSVDB-73751.yml

```diff
@@ -0,0 +1,11 @@
+---
+gem: spree
+osvdb: 73751
```

Comment:
The description on this OSVDB entry is non-existent at all, when they've got the updates working again are you going to submit a new description for the item?